gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/inspection/inspect.ts

```diff
@@ -126,13 +126,42 @@ export default class OCR2Inspect extends SolanaCommand {
     return buffer
   }
 
+  calculateOwedPayment = (
+    latestRoundId: number,
+    fromRoundId: number,
+    observationPaymentGjuels: number,
+    oraclePaymentGjuels: BN,
```

Comment:
Feels weird to accept `number` and `BN` in the same fn. Let's unify it with `number`